### PR TITLE
Env config user artifacts

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: pkg/cli/cvd.go [hunk 1]
```diff
@@ -159,6 +162,50 @@ func (c *cvdCreator) Create() ([]*hoapi.CVD, error) {
...
+	}
+	var b bytes.Buffer
+	err = tmpl.Execute(&b, uaEnvConfigTmplData{ArtifactsDir: d.ArtifactsDir})
+	if err != nil {
+		panic(err)
+	}
+	result := make(map[string]interface{})
+	if err := json.Unmarshal(b.Bytes(), &result); err != nil {
+		panic(err)
+	}
```
Comment (0405ysj) [inline pkg/cli/cvd.go:205]: Uniting the style for error handling? Former reuses `err` variable, but latter declares `err` variable inside conditional statement.

File: pkg/cli/cvd.go [hunk 2]
```diff
@@ -159,6 +162,50 @@ func (c *cvdCreator) Create() ([]*hoapi.CVD, error) {
...
+`
+
+type uaEnvConfigTmplData struct {
+	ArtifactsDir string
+}
+
+func (d *uaEnvConfigTmplData) Value() map[string]interface{} {
+	tmpl, err := template.New("").Parse(uaEnvConfigTmpl)
+	if err != nil {
+		panic(err)
```
Comment (0405ysj) [inline pkg/cli/cvd.go:195]: Why does this function use `panic` to handle error, though all callers of this function already does error propagation?

**Review by 0405ysj — APPROVED** [review]
Left some comments, but comments doesn't harm entire code design of this PR. Pressing approval in advance.

File: pkg/cli/cvd.go (hunk 2 above)
Comment (ser-io) [reply pkg/cli/cvd.go]: my thinking was that an error parsing the template should not be propagated, however the other two errors in this method should be propagated. I moved parsing the template into an `init` block.

**Pull request merged**